docs/core/option.md

```diff
@@ -67,7 +110,34 @@ Option<int> nonZeroLength = OnlyNonWhitespace("  hello  ")
     .Where(x => x.Length > 0);
 ```
 
-## FlatMap / SelectMany
+### OfType
+
+Use `OfType` to filter option value based on its type.
+
+> It is as a shortcut to `FlatMap(a => a is U u ? Some(u) : None)`.
```

Comment:
```suggestion
> This is essentially a short-hand for `FlatMap(a => a is U u ? Some(u) : None)`.
```

Reply:
Done (just changed "shot-hand" to "shorthand")